src/ontology/uberon-edit.obo

```diff
@@ -224685,6 +224685,20 @@ relationship: dc-contributor https://orcid.org/0000-0002-4341-3265
 relationship: dc-contributor https://orcid.org/0000-0003-2804-127X ! Sanjay Jain
 property_value: dcterms-date "2024-08-28T16:10:50Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600117
+name: antimesosalpinx
+def: "A posterior free border of a mesosalpinx." [https://vanat.ahc.umn.edu/ungDissect/Lab17/Lab17.html]
+synonym: "anti-mesosalpinx" EXACT [doi:10.1101/2020.08.23.263772]
+synonym: "posterior free border of mesosalpinx" EXACT [https://vanat.ahc.umn.edu/ungDissect/Lab17/Lab17.html]
```

Comment:
Everything looks good - only comment: I could not find the word 'posterior' in the reference